loader/loader.go

```diff
@@ -1035,7 +1029,7 @@ func (l *Loader) restoreData(ctx context.Context) error {
 		select {
 		case <-ctx.Done():
 			log.Infof("stop dispatch data file job because %v", ctx.Err())
-			break
+			return nil
```

Comment:
No need to `l.closeFileJobQueue()`?

Reply:
`l.closeFileJobQueue()` just set `fileJobQueueClosed` flag to `true` and close `fileJob` channel, in this scenario it is not needed, however calling it has no side effect.

We have not made the resource recycle regulation when we meet the context cancelation.
For one thing, if we always check the context before any resource access, we can always avoid invalid resource access. On the other side, active close channel, reset resource to nil is always ok.